### DISCUSSION TITLE
Feeds take a considerable amount of time to load

### DISCUSSION THREAD

**Discussion by omentic** [community/community#13205] [Other Feature Feedback, Questions, & Ideas]
It takes around three seconds on a fast university connection, if not longer, to load the front page feed. 

The current feed design also suffers from this too - it's better at around a second and a half (still not good though). But this wasn't always the case! There was some sort of change around half a year ago (maybe longer?) that greatly bumped up the latency - before that, I hadn't ever seen the loading octocat.

Although I don't actually know anything about GitHub's web stack, I suspect a lot of this latency comes from rendering the feed client-side. GitLab - which relies heavily on client-side rendering - used to be borderline-unusable on my previous 5.3 mbit/sec connection. This article illustrates it, kinda: https://carter.sande.duodecima.technology/javascript-page-navigation/

![molasses octocat](https://github.githubassets.com/images/mona-loading-dimmed.gif)

**Comment by jfullstackdev** [comment]
I'm not experiencing that on Google Chrome, maybe the computer setup too, low memory, firewall, antivirus, and the browser itself

**Reply by regs01** [reply]
No, that's a long server response. Takes almost 5 to 10 seconds.

**Comment by hamidrezamd1380** [comment]
https://realit.ir/

**Comment by agausmann** [comment]
A year and a half later, and still it takes 5 seconds to load... The old feed had no such issue

**Comment by med-ab** [comment]
am i the only one who feels this jumping mascot while loading is kinda annoying..

![68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f6d6f6e612d6c6f6164696e672d64696d6d65642e676966](https://github.com/user-attachments/assets/c345379c-42d2-471f-904e-93075b17ea1e)

**Comment by tonymet** [comment]
I did a test on mine and it averaged 4 seconds loading time.  I'm following 58 people, unsure on # of topics but it's probably similar.
